data_structures/stacks/evaluate_postfix_notations.py

```diff
@@ -1,12 +1,24 @@
-'''
```

Comment:
This is called a docstring which explains the intent of this file.  The docstring is used by [pydoc](https://docs.python.org/3/library/pydoc.html) if you run the command `python3 -m pydoc data_structures/stacks/evaluate_postfix_notations.py`

By converting this to a comment, this PR would remove that documentation.